### DISCUSSION TITLE
Column select in tmux

### DISCUSSION THREAD

**Discussion by thankph** [kovidgoyal/kitty#5310] [Q&A] [answered]
I'm having trouble getting column select to work in tmux.  

I can do mouse column select when not in tmux by using ctrl + alt keys.   In Alacritty, I'm able to do column select within tmux by holding down ctrl+shift keys.  I've tried various options in kitty.conf but I'm not able to solve this.   Really would appreciate if someone could point me in the right direction.   Thanks.

**Comment by page-down** [comment, marked as answer]
The default configuration works in ungrabbed mode, and tmux grabs the mouse when mouse mode is turned on, so the default configuration will not work.

https://sw.kovidgoyal.net/kitty/conf/#shortcut-kitty.Start-selecting-text-in-a-rectangle

The following configuration adds `grabbed,` to the modes and will make the respective mouse action work properly.

```conf
mouse_map ctrl+alt+left press grabbed,ungrabbed mouse_selection rectangle
```

**Reply by thankph** [reply]
Thank you, that resolved the issue!   I love the support you get from open source community.  I made a donation to the project.
